src/main/java/com/epam/bookstoreservice/mapper/TokenResponseDTOAssembler.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.bookstoreservice.mapper;
+
+import com.epam.bookstoreservice.dto.response.TokenResponseDTO;
+import com.epam.bookstoreservice.model.TokenModel;
+import org.springframework.hateoas.server.RepresentationModelAssembler;
+import org.springframework.stereotype.Component;
+
+/**
+ * model assembler for tokenResponseDTO
+ */
+@Component
+public class TokenResponseDTOAssembler implements RepresentationModelAssembler<TokenResponseDTO, TokenModel> {
```

Comment:
You could move this assembler to a new package called 'hateoas/assembler' since this feature is related to it.